pkg/webui/components/navigation/side/item/item.styl

```diff
@@ -118,3 +125,9 @@
 
   &-open
     transform: translateY(-50%) rotate(180deg)
+
+.fly-out-list
+  left: calc(3rem + 1px)
+  top: 0
+  transition: visibility $ad.s
```

Comment:
After trying this out, I have to say that your original solution (with the area around the fly-out) is actually preferable here, mainly because this introduces weird delays when hovering along the sidebar icons, wich would be quite hacky to circumvent.

So, let's revert this back. Mea culpa!